src/layouts/components/Explore.tsx

```diff
@@ -0,0 +1,135 @@
+import classNames from "classnames";
+import { useState } from "react";
+import { useTranslation } from "next-i18next";
+import { HeaderNavLinkItem } from "./HeaderNavLinkItem";
+
+export function Explore() {
+  const { t } = useTranslation("layout");
+  const entries: Array<{ title: string; subtitle: string }> = t(
+    "header.explore.labels",
+    { returnObjects: true }
+  );
+  const MenuItems = {
+    dropDownItems: {
+      label: t("header.explore.title"),
+      items: [
+        {
+          icon: "dfi",
+          href: "",
+          title: entries[0].title,
+          subtitle: entries[0].subtitle,
+        },
+        {
+          icon: "invest",
+          href: "",
+          title: entries[1].title,
+          subtitle: entries[1].subtitle,
+        },
+        {
+          icon: "dex",
+          href: "",
+          title: entries[2].title,
+          subtitle: entries[2].subtitle,
+        },
+        {
+          icon: "wallet",
+          href: "",
+          title: entries[3].title,
+          subtitle: entries[3].subtitle,
+        },
+        {
+          icon: "masternode",
+          href: "",
+          title: entries[4].title,
+          subtitle: entries[4].subtitle,
+        },
+      ],
+    },
+    image: {
+      title: t("header.explore.image.title"),
+      subtitle: t("header.explore.image.subtitle"),
+      href: "",
+    },
+  };
+
+  const [hoverState, setHoverState] = useState<string | undefined>(undefined);
+
+  return (
+    <div className="flex flex-col">
+      <div className="text-dark-500 font-bold leading-5 mb-[36px] md:block hidden">
+        {MenuItems.dropDownItems.label}
+      </div>
+      <div className="flex xl:flex-row md:flex-col gap-8">
+        <div className="flex md:flex-row flex-col gap-y-8 gap-x-6">
+          <div className="flex-col space-y-8 min-w-[368px]">
+            <HeaderNavLinkItem
```

Comment:
i think `Explore` has a similar UI as `Build` menu item, can we change the code here to match `Build` code structure?